lib.nix

```diff
@@ -15,7 +15,10 @@ let
         url = "${spec.url}/archive/${spec.rev}.tar.gz";
         inherit (spec) sha256;
       }) (removeAttrs iohkNixArgs ["iohkNixJsonOverride"]);
-  iohkNix       = mkIohkNix { application = "iohk-ops"; };
+  iohkNix       = mkIohkNix {
+    application = "iohk-ops";
+    nixpkgsJsonOverride = ./nixpkgs-src.json;
```

Comment:
still need to decide on what to do with this
without that present, half of iohk-ops obeys a different pin, and hydra caching fails
with this present, nixops fails to build, so nix-shell is broken